debian/rules

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/make -f
+
+export DEB_VERSION=$(shell dpkg-parsechangelog | egrep '^Version:' | cut -f 2 -d ' ')
+export BUILD_VERSION=v${DEB_VERSION}-debian-pragmatic
+
+
+%:
+	dh $@
+
+override_dh_usrlocal:
+	echo "nope"
+override_dh_systemd_start:
+	echo "Not running dh_systemd_start"
+override_dh_auto_clean:
+	rm -rf lua-cs-bouncer
+override_dh_auto_test:
+override_dh_auto_build:
+override_dh_auto_install:
+	mkdir -p debian/crowdsec-nginx-bouncer/etc/nginx/conf.d/
+	cp nginx/crowdsec_nginx.conf debian/crowdsec-nginx-bouncer/etc/nginx/conf.d/
+	git clone https://github.com/crowdsecurity/lua-cs-bouncer.git
+	mkdir -p debian/crowdsec-nginx-bouncer/usr/local/lua/crowdsec/
```

Comment:
please use /usr instead of /usr/local. If it requires some code changes, use a patch.